### ISSUE TITLE
Own tweets

### ISSUE THREAD

**Issue by marzman95** [FirstLegoLeague/twitter-admin#6]
Two features to handle the tweets that the account(s) of a tournament sends:

- Auto-approve those accounts: using a form/input to add those accounts, instead of waiting for the first tweet of such an account to appear in the system.
- View statistics of those tweets.

**Comment by poelstra** [comment]
Good idea. I always did this by manually adding them to the database :wink:
